NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
why not? I have seen other cases done that way on other projects at apache.

I believe we need something on the Notice.